File: API/Dtos/Product.cs
namespace API.Dtos
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public string Type { get; set; }
        public string Brand { get; set; }
        public string PictureUrl { get; set; }
        public string Publisher { get; set; }
        public int QuantityInStock { get; set; }
    }


}

File: client/src/app/models/product.ts
export interface Product {
  find(arg0: (i: any) => boolean): unknown;
  id: number;
  name: string;
  description: string;
  price: number;
  type: string;
  brand: string;
  pictureUrl: string;
  quantityInStock: number;
  //publisher: string;
}

export interface ProductParams {
  sort: string;
  search?: string;
  types: string[];
  brands: string[];
  page: number;
  pageSize: number;
}
